modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -113,19 +111,33 @@ To be prepared for the migration, both sides need to match the prerequisites. Pl
 ** See the xref:installation/apps_management_installation.adoc[Installing and Managing Apps] documentation for more details respecively the section xref:installing-required-components[Installing Required Components] below.
 
 * All users must have an email address and they must be unique. +
-The `occ migrate:to-kiteworks:verify` step will point out missing email addresses. These must be rectified before any migration can start.
+The `occ migration:verify` step will point out missing email addresses. These must be rectified before any migration can start.
+
+* We highly recommend installing and enabling, if not already present and enabled, the {oc-marketplace-url}/apps/impersonate[Impersonate] app. This app can be used to solve file and folder case conflicts that can be reported during the migration process.
+
+==== Installing Required Components
+
+* Before you install the migration app, you must install the custom `rclone` binary first. The migration app cannot be activated if the _correct_ rclone is missing. If you have installed rclone before, we recommend uninstalling and install the custom rclone binary instead. After installing the rclone binary and making it executable, check the output of:
++
+--
+[source,bash]
+----
+rclone help backends
+----
+
+If the correct binary is installed and executed, it shows a list including: `Kiteworks`.
```

Comment:
This is adding a lot of "new" text that I think was deleted recently.

Reply:
oh man 🤦‍♂️ what a mess 😢